packages/api/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@code-reviewer/api",
+  "version": "0.0.0",
+  "description": "Code Reviewer backend API",
+  "homepage": "https://github.com/TeamHarrington/code-reviewer/tree/master/packages/api#readme",
+  "license": "ISC",
+  "main": "index.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/TeamHarrington/code-reviewer.git"
+  },
+  "scripts": {
+    "start": "ts-node --project tsconfig.json index.ts"
+  },
+  "dependencies": {
+    "apollo-server-express": "^2.9.5",
+    "express": "^4.17.1",
+    "graphql": "^14.5.8",
+    "lodash": "^4.17.15"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.1",
+    "@types/lodash": "^4.14.141",
+    "@types/node": "^12.7.11",
+    "ts-node": "^8.4.1",
+    "typescript": "^3.6.3"
```

Comment:
Can we add `nodemon` as a dev dependency?

Reply:
Added